Failed building hpx-1.2.0-rc1 on Ubuntu16.04 x86-64 Virtualbox VM

## Expected Behavior

... building successful

## Actual Behavior

... building failed


## Steps to Reproduce the Problem

sudo apt-get install -y \\
     cmake \\
     libboost-all-dev \\
     libhwloc-dev \\
     google-perftools \\
     ;

wget https://github.com/STEllAR-GROUP/hpx/archive/1.2.0-rc1.tar.gz
tar xf 1.2.0-rc1.tar.gz
cd hpx-1.2.0-rc1

app_root=$(pwd)
mkdir bld obj
cd bld
cmake \\
    -DHPX_WITH_MALLOC=system \\
    -DCMAKE_INSTALL_PREFIX=${app_root}/obj \\
    ..
make




[  9%] Building CXX object src/CMakeFiles/hpx.dir/runtime/applier/bind_naming_wrappers.cpp.o
[  9%] Building CXX object src/CMakeFiles/hpx.dir/runtime/applier/applier.cpp.o
[  9%] Building CXX object src/CMakeFiles/hpx.dir/runtime/serialization/output_archive.cpp.o
[  9%] Building CXX object src/CMakeFiles/hpx.dir/runtime/serialization/detail/polymorphic_nonintrusive_factory.cpp.o
[ 12%] Building CXX object src/CMakeFiles/hpx.dir/runtime/serialization/detail/polymorphic_id_factory.cpp.o
[ 12%] Building CXX object src/CMakeFiles/hpx.dir/runtime/serialization/detail/polymorphic_intrusive_factory.cpp.o
[ 12%] Building CXX object src/CMakeFiles/hpx.dir/runtime/threads/scheduler_specific_ptr.cpp.o
[ 12%] Building CXX object src/CMakeFiles/hpx.dir/runtime/threads/thread_pool_base.cpp.o
[ 12%] Building CXX object src/CMakeFiles/hpx.dir/runtime/threads/topology.cpp.o
/home/aaa/hpx/hpx-1.2.0-rc1/src/runtime/threads/topology.cpp: In member function ‘int hpx::threads::topology::get_numa_domain(const void*) const’:
/home/aaa/hpx/hpx-1.2.0-rc1/src/runtime/threads/topology.cpp:1332:13: error: ‘HWLOC_MEMBIND_BYNODESET’ was not declared in this scope
             HWLOC_MEMBIND_BYNODESET);
             ^
/home/aaa/hpx/hpx-1.2.0-rc1/src/runtime/threads/topology.cpp:1332:36: error: ‘hwloc_get_area_memlocation’ was not declared in this scope
             HWLOC_MEMBIND_BYNODESET);
                                    ^
src/CMakeFiles/hpx.dir/build.make:1622: recipe for target 'src/CMakeFiles/hpx.dir/runtime/threads/topology.cpp.o' failed
make[2]: *** [src/CMakeFiles/hpx.dir/runtime/threads/topology.cpp.o] Error 1
CMakeFiles/Makefile2:1897: recipe for target 'src/CMakeFiles/hpx.dir/all' failed
make[1]: *** [src/CMakeFiles/hpx.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2


## Specifications

... Please describe your environment

  - HPX Version: hpx-1.2.0-rc1
  - Platform (compiler, OS): gcc (Ubuntu 5.4.0-6ubuntu1~16.04.10) 5.4.0 20160609

--- a/src/runtime/threads/topology.cpp
+++ b/src/runtime/threads/topology.cpp
@@ -1317,7 +1317,7 @@ namespace hpx { namespace threads
 
     int topology::get_numa_domain(const void *addr) const
     {
-#if HWLOC_API_VERSION >= 0x00010b00
+#if HWLOC_API_VERSION >= 0x00010b06
         hpx_hwloc_bitmap_wrapper *nodeset = topology::bitmap_storage_.get();
         if (nullptr == nodeset)
         {

